src/ck-cp/pathHistory.C

```diff
@@ -9,7 +9,7 @@
 
 //#define DEBUG  1
 #include "PathHistory.decl.h"
-#include "LBDatabase.h"
+#include "LBManager.h"
```

Comment:
For all these places where you are just changing the header name, is the header even needed? There are no other changes in the file so it seems like LBDatabase and LBManager aren't even used by these files?